index.html

```diff
@@ -5,8 +5,13 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
         <!-- Bootstrap CSS -->
+<<<<<<< HEAD
```

Comment:
hi Sam, please resolve this conflicts manually

Reply:
Ok this is done as per earlier :)